src/annotator/annotation-sync.js

```diff
@@ -28,11 +28,11 @@
 export class AnnotationSync {
   /**
    * @param {EventBus} eventBus - Event bus for communicating with the annotator code (eg. the Guest)
-   * @param {SidebarBridge} bridge - Channel for communicating with the sidebar
+   * @param {SidebarBridge} sidebarBridge - Channel for communicating with the sidebar
```

Comment:
We might want to rename the `Bridge` class in future to better align it with the `<frameRole>RPC` field names and `PortRPC` class that exists. Or perhaps even combine it with `PortRPC`, maybe only for certain uses? The main distinction with `PortRPC` in the current code is that a `Bridge` can be connected to zero or many ports, whereas a `PortRPC` is always connected to exactly one.

Reply:
It makes sense.